.github/workflows/dev.yaml

```
@@ -57,25 +57,24 @@ jobs:
       - name: Publish dev images
         id: publish_images
         run: |
-          BAKE_OUTPUT=$(bake --product "${{ matrix.product }}" --image-version 0.0.0-dev --shard-count "${{matrix.shard_count}}" --shard-index "${{matrix.shard_index}}" --push 2>&1)
-          # echo "$BAKE_OUTPUT"
-          # needed for multiline strings in github actions
-          # EOF=$(dd if=/dev/urandom bs=15 count=1 status=none | base64)
-          # {
-          #   echo "PUSHED_MANIFESTS<<$EOF";
-          #   # find all pushed manifests based on the output of the bake command
-          #   # this is needed for image signing, since images should be referenced by their digest
-          #   echo "$BAKE_OUTPUT" | awk '/^#[0-9]+ pushing manifest for docker.stackable.tech\/stackable\/${{ matrix.product }}:.+ done$/ { print $5 }' >> "$GITHUB_OUTPUT";
-          #   echo "$EOF" >> "$GITHUB_OUTPUT";
-          # } >> "$GITHUB_OUTPUT"
-      # - name: Set up Cosign
-      #   uses: sigstore/cosign-installer@v3.1.2
-      # - name: Sign images
-      #   run: |
-      #     while IFS= read -r MANIFEST; do
-      #       echo "signing $MANIFEST";
-      #       cosign sign -y "$MANIFEST";
-      #     done <<< "${{steps.publish_images.outputs.PUSHED_MANIFESTS}}"
+          # bake images and load them to local docker repo for signing
+          # TODO: buildx cannot --load and --push at the same time
+          bake --product "${{ matrix.product }}" \
+          --image-version 0.0.0-dev \
+          --shard-count "${{matrix.shard_count}}" \
+          --shard-index "${{matrix.shard_index}}" \
+          --export-tags-file bake-target-tags
+          # push images to image repository
+          if [ -f bake-target-tags ]; then
+            docker image push < bake-target-tags
+          fi
+      - name: Set up Cosign
+        uses: sigstore/cosign-installer@v3.1.2
+      - name: Sign images
+        run: |
+          if [ -f bake-target-tags ]; then
+            cosign sign -y < bake-target-tags
```

`cosign sign` doesn't read the list of tags from stdin.

```suggestion
            cosign sign -y "$(< bake-target-tags)"
```